documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,250 @@ You may obtain a copy of the License at
-to a Stream is sent to exactly one Reader within the ReaderGroup.  There could
-be 1 Reader in the ReaderGroup, there could be many.  There could be many
-different ReaderGroups simultaneously reading from any given Stream.
+name of the Reader Group associated with it. Pravega guarantees that each Event published
+to a Stream is sent to exactly one Reader within the Reader Group. There could
+be one or more Readers in the Reader Group and there could be many different Reader Groups simultaneously reading from any given Stream.
 
-You can think of a ReaderGroup as a "composite Reader" or a "distributed
+A Reader Group can be considered as a "composite Reader" or "distributed
 Reader", that allows a distributed application to read and process Stream data
-in parallel, such that a massive amount of Stream data can be consumed by a
-coordinated fleet of Readers in a ReaderGroup.  A collection of Flink tasks
-processing Stream data in parallel is a good example use of a ReaderGroup.
+in parallel. A large amount of Stream data can be consumed by a coordinated group of Readers in a Reader Group.  For example, a collection of Flink tasks processing Stream data in parallel using Reader Group.
 
-For more details on the basics of working with Pravega Readers and Writers,
-see [Working with Pravega: Basic Reader and
+For more details on the basics of working with Pravega Readers and Writers, please see [Working with Pravega: Basic Reader and
 Writer](basic-reader-and-writer.md#working-with-pravega-basic-reader-and-writer).
 
-We need to talk in more detail about the relationship between Readers,
-ReaderGroups and Streams and the ordering guarantees provided by Pravega.  But
-first, we need to describe what a Stream Segment is.
-
 ## Stream Segments
 
-A Stream is decomposed into a set of Stream Segments; a Stream Segment is a
-shard or partition of a Stream.
+A Stream is decomposed into a set of Segments generally referred as **Stream Segments**; a Stream Segment is a shard or partition of a Stream.
 
 ![Stream Segment](img/stream.segment.png) 
 
-### An Event is Stored within a Stream Segment
+### Events in a Stream Segment
 
-
-The Stream Segment is the container for Events within the Stream.   When an
+The Stream Segments acts as a container for Events within the Stream. When an
 Event is written into a Stream, it is stored in one of the Stream Segments based
-on the Event's Routing Key.  Pravega uses consistent hashing to assign Events to
-Stream Segments. Event Routing Keys are hashed to form a "key space" .  The key
+on the Event's Routing Key. Pravega uses consistent hashing to assign Events to
+Stream Segments. Event Routing Keys are hashed to form a "key space". The key
 space is then divided into a number of partitions, corresponding to the number
-of Stream Segments. Consistent hashing determines which Segment an Event is
-assigned to.
+of Stream Segments. Consistent hashing determines the allotment of the specific Segment to an Event.
```

Comment:
Please, change: `the allotment of Events to Stream Segments.`

Reply:
done